shared/constants/chat2/meta.js

```diff
@@ -308,3 +319,14 @@ export const findConversationFromParticipants = (state: TypedState, participants
       meta.teamType === 'adhoc' && meta.participants.toSet().equals(toFind)
   )
 }
+
+export const getConversationRetentionPolicy = (
+  state: TypedState,
+  conversationIDKey: Types.ConversationIDKey
+) => {
+  const conv = state.chat2.metaMap.get(conversationIDKey)
```

Comment:
just use Constants.getMeta(conversationIDKey) which returns the blank meta if nothing is there